app/views/pages/cookies.html.erb

```diff
@@ -0,0 +1,161 @@
+<div class="govuk-grid-row">
+  <div class="govuk-grid-column-two-thirds">
+    <h1 class="govuk-heading-xl">
+      Cookies
+    </h1>
+
+    <p class="govuk-body-l">
+      TDLR about cookies
+    </p>
+
+    <p class="govuk-body">
+    Cookies are used to:
+    </p>
+
+    <ul class="govuk-list govuk-list--bullet">
+      <li>measure how you use the this service so it can be updated and improved based on your needs</li>
+      <li>remember the notifications you've seen so that we don't show them to you again</li>
+      <li>remember who you are when you have logged in so we can show you your information and information relevant to you</li>
+    </ul>
+
+    <div class="panel panel-border-wide">
+      <p class="govuk-body">Teachers: claim back your student loan repayments cookies aren't used to identify you personally.</p>
+    </div>
+
+    <p class="govuk-body">
+      Find out more about <a class="govuk-link" href="http://www.aboutcookies.org/">how to manage cookies.</a>
+    </p>
+
+    <h2 class="govuk-heading-l">
+      How we use cookies
+    </h2>
+
+    <h3 class="govuk-heading-m">
+      Measuring website usage (Google Analytics)
```

Comment:
We don't use GA, I don't think.

Reply:
changed to XXXX I think we will need to use something though.